### PR TITLE
adds validate option to template

### PR DESCRIPTION
**Pull request by bcoca**
requires the previous pr to copy module as it just passes it through and lets the copy module do the work

### CODE REVIEW

File: library/copy [hunk 1]
```diff
@@ -74,6 +79,8 @@ examples:
      description: "Example from Ansible Playbooks"
    - code: "copy: src=/mine/ntp.conf dest=/etc/ntp.conf owner=root group=root mode=644 backup=yes"
      description: "Copy a new C(ntp.conf) file into place, backing up the original if it differs from the copied version"
+   - code: "copy: src=/mine/sudoers dest=/etc/sudoers validate='visudo -c %s'
+     description: "Copy a new C(sudoers) file into place, after passing validation with visdo"
```
Comment (sejo) [inline library/copy:83]: typo should be visudo not visdo :P

Comment (bcoca) [inline library/copy:83]: You are hired as my personal spellchecker!

Will resubmit PR with typo fix.

## 

Brian Coca
Stultorum infinitus est numerus
0110000101110010011001010110111000100111011101000010000001111001011011110111010100100000011100110110110101100001011100100111010000100001
Pedo mellon a minno

**Comment by mpdehaan** [comment]
past our Monday feature freeze, so in queue for consideration in 1.2.   Thanks!

**Comment by mpdehaan** [comment]
The filecheck option to template seems somewhat odd here, can you put the visudo example in here instead.  If so, I can merge this.

Thanks!

**Comment by mpdehaan** [comment]
will let you resubmit when this has a real example using visudo versus the filecheck.

**Pull request closed without merging**